quantum/stm32/STM32F1xx/chconf.h

```diff
@@ -0,0 +1,524 @@
+/*
```

Comment:
chconf, halconf, and mcuconf should really be keyboard specific. In certain cases, so will board.c.
Different keyboards can use different channels, etc. and forcing everything to use the QMK defaults could force suboptimal pin assignment

I'm ok w/ these as defaults but they should never be chosen if they are present for keyboards.

Reply:
I can make that a hard requirement, "Provide sane defaults, but allow keyboards to override". Ideally I would like it to behave consistently with Proton C, rather than a bootstrap for any f103 keyboard (however if it provides some or all of that functionality then thats also a win).

Reply:
I have just tested locally and chconf, halconf, and mcuconf can be added to the keyboard directory and they take precedence over the defaults.